src/main/resources/templates/db_scripts/setPreviouslyHeldDefault.sql

```diff
@@ -0,0 +1,11 @@
+START TRANSACTION;
+
+ALTER TABLE ${myuniversity}_${mymodule}.instance DISABLE TRIGGER USER;
+
+UPDATE ${myuniversity}_${mymodule}.instance
+  SET jsonb = JSONB_SET(instance.jsonb, '{previouslyHeld}', TO_JSONB(false))
+WHERE jsonb->>'previouslyHeld' IS NULL;
+
+ALTER TABLE ${myuniversity}_${mymodule}.instance ENABLE TRIGGER USER;
+
+END TRANSACTION;
```

Comment:
I think it is a good idea to have a test for the upgrade scripts as well. For example, `ItemEffectiveCallNumberDataUpgradeTest` does something similar for the setting of effective call number components for existing records.

Reply:
I tried to write this, but I can't figure out the syntax for the database command to set the previouslyHeld field back to null in perparation for running the upgrade script.  I either get incomprehensible error messages about polymorphism or syntax errors.  I've posted what I have, the test is in PreviouslyHeldDataUpgradeTest.java

Reply:
Gotcha, As the API has effectively already done the defaulting of any new records, they would need to be reset.

I hadn't thought of that :-/